From a07a005cd681e8014600e0ff0524b7fb508202f6 Mon Sep 17 00:00:00 2001
From: "bo.yang" <bo.yang@amlogic.com>
Date: Mon, 12 Jan 2015 11:29:39 +0800
Subject: [PATCH 5321/5965] PD#101206: reorder ao pinmux index and update n300
 1g/1.5g dtd

reorder ao pinmux from 10 to 13. and pinmux index 11 to 10.

the following modules dtd are changed:
1. ao uart
2. i2c b
3. remote
---
 .../boot/dts/amlogic/mesong9tv_n300_15g.dtd   | 20 +++++++++++--------
 .../boot/dts/amlogic/mesong9tv_n300_1g.dtd    | 20 +++++++++++--------
 2 files changed, 24 insertions(+), 16 deletions(-)

diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd
index 6c8c3b60a167..1350343f8440 100755
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n300_15g.dtd
@@ -1095,7 +1095,7 @@
 			amlogic,clrmask = <3 0x10
 					   5 0x2000000
 					   9 0x1
-					   10 0x100000>;
+					   10 0x140000>;
 			amlogic,pins = "GPIOY_8";
 		};
 
@@ -1103,7 +1103,8 @@
 		//$$ L2 PROP_U32 2 = "amlogic,setmask"
 		//$$ L2 PROP_STR 2 = "amlogic,pins"
 		ao_uart_pins:ao_uart {
-			amlogic,setmask = <10 0x1800>;
+			amlogic,setmask = <13 0x1800>;
+			amlogic,clrmask = <13 0x06000000>;
 			amlogic,pins = "GPIOAO_0", "GPIOAO_1";
 		};
 
@@ -1111,8 +1112,11 @@
 		//$$ L2 PROP_U32 2 = "amlogic,setmask"
 		//$$ L2 PROP_STR 4 = "amlogic,pins"
 		a_uart_pins:a_uart {
-			amlogic,setmask=<4 0x3c00>;
-			amlogic,pins="GPIOX_12", "GPIOX_13", "GPIOX_14", "GPIOX_15";
+			amlogic,setmask = <4 0x3c00>;
+			amlogic,clrmask = <0 0x44
+					   3 0x880
+					   7 0xf000>;
+			amlogic,pins = "GPIOX_12", "GPIOX_13", "GPIOX_14", "GPIOX_15";
 		};
 
 		//$$ MATCH "uart_1_pin_match" = "&b_uart_pins"
@@ -1352,9 +1356,9 @@
 		//$$ L2 PROP_U32 2 = "amlogic,setmask"
 		//$$ L2 PROP_STR 2 = "amlogic,pins"
 		b_i2c_master:b_i2c {
-			amlogic,setmask=<11 0xc000>;
-			amlogic,clrmask=<9 0xf00000c0>;
-			amlogic,pins="GPIOY_12","GPIOY_13";
+			amlogic,setmask = <10 0xc000>;
+			amlogic,clrmask = <9 0xf00000c0>;
+			amlogic,pins = "GPIOY_12","GPIOY_13";
 
 		};
 
@@ -1362,7 +1366,7 @@
 		//$$ L2 PROP_U32 2 = "amlogic,setmask"
 		//$$ L2 PROP_STR = "amlogic,pins"
 		remote_pins:remote_pin {
-			amlogic,setmask = <10 0x1>;
+			amlogic,setmask = <13 0x1>;
 			amlogic,pins = "GPIOAO_7";
 		};
 
diff --git a/arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd b/arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd
index 4210f9a76342..84b1587e5caa 100755
--- a/arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd
+++ b/arch/arm/boot/dts/amlogic/mesong9tv_n300_1g.dtd
@@ -1095,7 +1095,7 @@
 			amlogic,clrmask = <3 0x10
 					   5 0x2000000
 					   9 0x1
-					   10 0x100000>;
+					   10 0x140000>;
 			amlogic,pins = "GPIOY_8";
 		};
 
@@ -1103,7 +1103,8 @@
 		//$$ L2 PROP_U32 2 = "amlogic,setmask"
 		//$$ L2 PROP_STR 2 = "amlogic,pins"
 		ao_uart_pins:ao_uart {
-			amlogic,setmask = <10 0x1800>;
+			amlogic,setmask = <13 0x1800>;
+			amlogic,clrmask = <13 0x06000000>;
 			amlogic,pins = "GPIOAO_0", "GPIOAO_1";
 		};
 
@@ -1111,8 +1112,11 @@
 		//$$ L2 PROP_U32 2 = "amlogic,setmask"
 		//$$ L2 PROP_STR 4 = "amlogic,pins"
 		a_uart_pins:a_uart {
-			amlogic,setmask=<4 0x3c00>;
-			amlogic,pins="GPIOX_12", "GPIOX_13", "GPIOX_14", "GPIOX_15";
+			amlogic,setmask = <4 0x3c00>;
+			amlogic,clrmask = <0 0x44
+					   3 0x880
+					   7 0xf000>;
+			amlogic,pins = "GPIOX_12", "GPIOX_13", "GPIOX_14", "GPIOX_15";
 		};
 
 		//$$ MATCH "uart_1_pin_match" = "&b_uart_pins"
@@ -1352,9 +1356,9 @@
 		//$$ L2 PROP_U32 2 = "amlogic,setmask"
 		//$$ L2 PROP_STR 2 = "amlogic,pins"
 		b_i2c_master:b_i2c {
-			amlogic,setmask=<11 0xc000>;
-			amlogic,clrmask=<9 0xf00000c0>;
-			amlogic,pins="GPIOY_12","GPIOY_13";
+			amlogic,setmask = <10 0xc000>;
+			amlogic,clrmask = <9 0xf00000c0>;
+			amlogic,pins = "GPIOY_12","GPIOY_13";
 
 		};
 
@@ -1362,7 +1366,7 @@
 		//$$ L2 PROP_U32 2 = "amlogic,setmask"
 		//$$ L2 PROP_STR = "amlogic,pins"
 		remote_pins:remote_pin {
-			amlogic,setmask = <10 0x1>;
+			amlogic,setmask = <13 0x1>;
 			amlogic,pins = "GPIOAO_7";
 		};
 
-- 
2.19.0

